VoxelSniperCore/src/main/java/com/github/kevindagame/VoxelBrushManager.java

```diff
@@ -154,23 +145,29 @@ public int registeredSniperBrushes() {
     /**
      * @return Amount of handles registered with the system under Sniper visibility.
      */
-    public int registeredSniperBrushHandles() {
-        return brushes.size();
+    public long registeredSniperBrushHandles() {
+        return brushes.values().stream().distinct().count();
```

Comment:
I'd personally prefer it if this were to be casted to an int, since there is no reason for this to be a long except that the count() method returns it